### PR TITLE
Fixing output issue with config:get undefined

### PR DESCRIPTION
**Pull request by ransombriggs**

### CODE REVIEW

**Comment by ransombriggs** [comment]
@dickeyxxx please review

**Comment by dickeyxxx** [comment]
I already submitted this https://github.com/heroku/heroku-apps/pull/20

**Comment by dickeyxxx** [comment]
but I suppose I didn't deal with the shell case

File: commands/config/get.js [hunk 1]
```diff
@@ -6,12 +6,16 @@ let co          = require('co');
 
 function* run (context, heroku) {
   let configVars = yield heroku.request({path: `/apps/${context.app}/config-vars`});
-  if (context.flags.shell) {
-    let v = configVars[context.args.key];
-    v = process.stdout.isTTY ? shellescape([v]) : v;
-    cli.log(`${context.args.key}=${v}`);
+  let v = configVars[context.args.key];
+  if (typeof(v) === 'undefined') {
+    cli.log(''); // match v2 output for missing
```
Comment (dickeyxxx) [inline commands/config/get.js]: I think you mean v3

File: commands/config/get.js [hunk 2]
```diff
@@ -6,12 +6,16 @@ let co          = require('co');
 
 function* run (context, heroku) {
   let configVars = yield heroku.request({path: `/apps/${context.app}/config-vars`});
-  if (context.flags.shell) {
-    let v = configVars[context.args.key];
-    v = process.stdout.isTTY ? shellescape([v]) : v;
-    cli.log(`${context.args.key}=${v}`);
+  let v = configVars[context.args.key];
+  if (typeof(v) === 'undefined') {
```
Comment (dickeyxxx) [inline commands/config/get.js]: why not `v === undefined`?

**Pull request merged**